### ISSUE TITLE
Error on creating initial project

### ISSUE THREAD

**Issue by bspeice** [brillout/vite-plugin-ssr#524]
### Description

I'm getting an error stack on creating a new project (`npm init vite-plugin-ssr@latest; cd <project>; npm install; npm run dev`, using the react-ts starter). `npm run prod` works as expected.

Happy to provide additional details as necessary.

### Error Stack

```shell
createErrorWithCleanStackTrace http://localhost:3000/node_modules/vite-plugin-ssr/dist/esm/utils/createErrorWithCleanStackTrace.js?v=4a979f34:7
    assert http://localhost:3000/node_modules/vite-plugin-ssr/dist/esm/utils/assert.js?v=4a979f34:27
    getPageContextSerializedInHtml http://localhost:3000/node_modules/vite-plugin-ssr/dist/esm/client/getPageContextSerializedInHtml.js?v=4a979f34:7
    getPageContext http://localhost:3000/node_modules/vite-plugin-ssr/dist/esm/client/getPageContext.js?v=4a979f34:9
    hydrate http://localhost:3000/@fs/var/home/bspeice/Development/ssr-blog/node_modules/vite-plugin-ssr/dist/esm/client/entry.js:11
    <anonymous> http://localhost:3000/@fs/var/home/bspeice/Development/ssr-blog/node_modules/vite-plugin-ssr/dist/esm/client/entry.js:8
createErrorWithCleanStackTrace.js:7:15
```

**Comment by brillout** [comment]
I can't reproduce using Chrome and Node.js 18

**Comment by bspeice** [comment]
Ah, that'll do it. I'm normally on Firefox, but this works fine if I switch to Chrome (also on Node 18). Let me see if I can't get more detail on what's breaking down in Firefox.

**Comment by bspeice** [comment]
Seems like a browser compatibility problem; the initial HTML index is served correctly, but Firefox can't retrieve the `vite-plugin-ssr_pageContext` script by element ID:

```js
>>> document.getElementById('vite-plugin-ssr_pageContext')
null 
```

...and thus triggers the exception. Chrome is able to fetch it just fine, I'm fine using that for dev builds. Not sure if you want to leave this open to address generally.

**Comment by brillout** [comment]
Indeed. Thanks.

Fix released in `0.4.54`.

In case your company is up for it: https://github.com/sponsors/brillout.
